special-cases/thrid-party-fraud-provider.html

```diff
@@ -0,0 +1,186 @@
+         const expiry = Airwallex.createElement("expiry");
+         const cvc = Airwallex.createElement("cvc");
+         cardNumberElementInstance = cardNumber;
+ 
+         // STEP #2a: mount cardNumberElement, expriyElement,cvcElement into the doms in web
+         cardNumber.mount("cardNumber");
+         expiry.mount("expiry");
+         cvc.mount("cvc");
+       } catch (error) {
+         document.getElementById("loading").style.display = "none";
+         document.getElementById("error").style.display = "block";
+         document.getElementById("error").innerHTML = error.message;
+         console.error("There was an error", error);
+       }
+ 
+       const elementsReady = {
+         cardNumber: false,
+         expiry: false,
+         cvc: false
+       };
+       // STEP #2b: Add an event listener to ensure the element is mounted
+       const cardNumberElement = document.getElementById("cardNumber");
+       const expiryElement = document.getElementById("expiry");
+       const cvcElement = document.getElementById("cvc");
+       const domElementArray = [cardNumberElement, expiryElement, cvcElement];
+ 
+       domElementArray.forEach((element) => {
+         element.addEventListener("onReady", (event) => {
+           const { type } = event.detail;
+           if (elementsReady.hasOwnProperty(type)) {
+             elementsReady[type] = true;
+           }
+ 
+           if (!Object.values(elementsReady).includes(false)) {
+             document.getElementById("loading").style.display = "none";
+             document.getElementById("element").style.display = "block";
+           }
+         });
+       });
+ 
+       const elementsCompleted = {
+         cardNumber: false,
+         expiry: false,
+         cvc: false
+       };
+ 
+       // Step #2c: Add an event listener to ensure all element is completed
+       domElementArray.forEach((element) => {
+         element.addEventListener("onChange", (event) => {
+           const { type, complete } = event.detail;
+           if (elementsCompleted.hasOwnProperty(type)) {
+             elementsCompleted[type] = complete;
+           }
+ 
+           const allElementsCompleted = !Object.values(
+             elementsCompleted
+           ).includes(false);
+           document.getElementById("submit").disabled = !allElementsCompleted;
+         });
+       });
+ 
+       // STEP #3: Provide payment details and click pay
+       document.getElementById("submit").addEventListener("click", () => {
+         // STEP #4: Froze the page in case shopper change again
+         document.getElementById("loading").style.display = "block";
+         document.getElementById("submit").disabled = true;
+ 
+         // STEP #5: Send request to your backend to get customer id and customer client secret
+         // const customerId = 'MOCK_CUSTOMER_ID';
+         // const customerClientSecret = 'MOCK_CUSTOMER_CLIENT_SECRET';
+         const customerId = "cus_hkstpf27fgk7ix5zd9q";
```

Comment:
do you think we should not put the real customer id here. just use "replace with your customer id"